docs/cluster-deployment/deploy-to-cluster-marketplace/deploy-to-cluster-aws/configure-aws.rst

```diff
@@ -145,6 +139,10 @@ Console`_.
 Deploy and connect
 ==================
 
+A popup menu will remind you of the username and password you selected for
```

Comment:
One minor detail - perhaps we could say "A popup menu will show you the username and password ...". "remind" seems like a strange word here. Not fussed about it though.

Reply:
'Remind' because you filled it out just two pages earlier, so it's not like a huge span of time has passed between the two 😁